#LRS Range
uint16 IdxR
float32 ran1
float32 ran2
float32 ran3

#LRS Intensity
uint16 IdxI
float32 intens1
float32 intens2
float32 intens3